### ISSUE TITLE
Tab section incorrectly parsed as chords and lyrics

### ISSUE THREAD

**Issue by beltonk** [martijnversluis/ChordSheetJS#1189]
I recently switched from HtmlTableFormatter to HtmlDivFormatter, and seems there are some logical different, e.g. sot in HtmlDivFormatter renders .lyrics but not .literals? I'm not sure about other difference, just feel some nuances.

**Comment by martijnversluis** [comment]
Hey @beltonk. What chord sheet and ChordSheetJS are you using?

I ran a small test:

```js
import { ChordProParser, HtmlDivFormatter } from "https://esm.sh/chordsheetjs";

const chordProTestSong = `
{sot}
One
Two
Three
{eot}`;

const song = new ChordProParser().parse(chordProTestSong);
const output = new HtmlDivFormatter().format(song);
console.log(output);
```

The output is:

```html
<div class='chord-sheet'>
  <div class='paragraph tab'>
    <div class='row'>
      <h3 class='label'>null</h3>
      <div class='literal'>
        One<br>
        Two<br>
        Three
      </div>
    </div>
  </div>
</div>
```

So it renders a `.literal`. I do spot a bug when rendering a `null` label.

**Comment by beltonk** [comment]
My fault... I assumed comment & literal are under columns, and mistakenly did some sanitization.
I have fixed my problems.
Please close this case.

**Comment by beltonk** [comment]
Close case, as the issue is not related to this library

**Comment by beltonk** [comment]
I can reproduce the issue, whenever there is any other {c: }, {soc}, {sov} directives in the ChordPro, the tab will be come lyrics lines instead of literal, no matter where these tags are.

```
const chordProTestSong = `
{sot}
One
Two
Three
{eot}
{c:Verse}
`;
```

**Comment by martijnversluis** [comment]
@beltonk I published `10.2.0` containing a bugfix that should resolve your issue. Please let me know if it does the job 👍
